src/components/RightSidebar/SharedItems/SharedItemsBrowser/SharedItemsBrowser.vue

```diff
@@ -0,0 +1,169 @@
+<!--
+  - @copyright Copyright (c) 2022 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<Modal size="large" v-on="$listeners">
```

Comment:
this feels really small for "large"
Height seems hardcoded to 700px and width to 900px? I would at least want the 90% height which is defined as max-height.
The width is okay for the file list and if we only want to show 3 columns of media, but I would also be fine with 5 or something if the screen is fullhd

Reply:
Height is no longer hard coded with the latest vue release...

Reply:
To which we can't update until the final for 24 is out because of the many breaking changes

Reply:
Yes, the breaking change was that the height is no longer fixed...

Reply:
something for the future @nickvergessen ?